servers/lib/tsconfig.json

```diff
@@ -4,28 +4,35 @@
     "baseUrl": "./",
     "declaration": true, //generate declaration '.d.ts' files
     "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
     "experimentalDecorators": true, //allow experimental decorators for es7
     "forceConsistentCasingInFileNames": false,
     "incremental": true,
-    "module": "commonjs", //use node module system
+    "lib": [
+      "ES2022"
+    ],
+    "paths": {
+      "@/*": ["src/*"]
+    },
+    "module": "ES2022", //use node module system
     "moduleResolution": "node", //use node module resolution strategy node
-    //"noImplicitAny": true, //raise error on any type
     "noImplicitReturns": true, //raise error on implicit returns
     "noUnusedLocals": true, //raise error on unused locals
     "noUnusedParameters": true, //raise no error on unused parameters
     "noFallthroughCasesInSwitch": false,
-    "outDir": "./dist",
+    "outDir": "dist",
     "removeComments": true, //remove comments from build
     "skipLibCheck": true,
     "sourceMap": true, //generate .map files
+    // "strict": true, //enable all strict type-checking options
     "strictBindCallApply": false,
-    //"strictNullChecks": true,
-    //"strict": true, //enable all strict type-checking options
-    "target": "es2022",
+    "target": "ES2022",
     "types": ["node", "jest"], //use node and jest types
     "typeRoots": [
       "node_modules/@types" //use node_modules/@types for type definitions
-    ]
+    ],
+    "allowJs": true,
```

Comment:
How does this setting help? Is it with imports inside tests? If the intention is to include JS code as well, please do add [checkJs](https://www.typescriptlang.org/tsconfig/#checkJs) setting as well.